src/store/slices/stakingHub/initialState.ts

```diff
@@ -29,6 +29,13 @@ type InitialState = {
     data: SubgraphParsedOutput;
     isFetching: boolean;
   };
+  config: {
+    needsUpdate: {
+      data: boolean;
```

Comment:
what is the use for data? i think we can remove right?

Reply:
it keeps the data/isFetching convention  that we have around